projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="range-wrap">
+  <div class="range-slider">
+    <div class="min-max-slider" #minMaxSlider>
+      <input id="min" class="min range-slider-range" name="min" type="range" min="0" max="1000" #minSlider appCypressData="min-range"/>
```

Comment:
I don't like these ids. Do you need them?

Reply:
I'm confused with all these class, id and cypresData values. Can you unify them a bit?

Reply:
done